tests/e2e/utils/pages/setup-mc/step-1-set-up-accounts.js

```diff
@@ -428,4 +419,15 @@ export default class SetUpAccountsPage extends MockRequests {
 	getAdsAccountConnectedText() {
 		return this.getGoogleAdsAccountCard().getByText( 'Connected' );
 	}
+
+	/**
+	 * Get Ads account connected notice text.
+	 *
+	 * @return {import('@playwright/test').Locator} Get ads account connected text.
```

Comment:
It could be more consistent a little bit.

```suggestion
	 * @return {import('@playwright/test').Locator} Get ads account connected notice text.
```